.travis.yml

```diff
@@ -13,3 +13,4 @@ env:
   global:
```

Comment:
why is this in github?

there should be private travis ci settings that we can define in our travis account and then set these. this look like (please correct me if I'm wrong) that we are sharing both our hostname and our secure token with the world?